app/src/main/java/a/a/a/yq.java

```diff
@@ -840,8 +840,9 @@ public void b(hC hCVar, eC eCVar, iC iCVar, boolean z2) {
         a2.add(new SrcCodeBean("FileUtil.java", Lx.e(e)));
         a2.add(new SrcCodeBean("RequestNetwork.java", Lx.h(e)));
         a2.add(new SrcCodeBean("RequestNetworkController.java", Lx.g(e)));
-        a2.add(new SrcCodeBean("BluetoothConnect.java", Lx.b(e)));
-        a2.add(new SrcCodeBean("BluetoothController.java", Lx.c(e)));
+        // unnecessary files when exporting project to android studio
+        // a2.add(new SrcCodeBean("BluetoothConnect.java", Lx.b(e)));
+        // a2.add(new SrcCodeBean("BluetoothController.java", Lx.c(e)));
```

Comment:
There's no if condition there to check whether Bluetooth is used in the project or not, so probably no need to export them because no one uses Bluetooth in sketchware lol

Reply:
they are not ```alpha```, only SwipeToRefresh library is alpha